.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: Pipeline
+
+on:
+  push:
+    branches:
+      - test_task
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      registry:
+        description: "Registry"
+        required: false
+        default: "ghcr.io"
+        type: string
+      server_address:
+        description: "Server address"
+        required: false
+        default: "smtp.gmail.com"
+        type: string
+
+env:
+  GH_TOKEN: ${{ github.token }}
```

Comment:
Do you really need to pass it as env? You can just use `{{ github.token }}` in the place where it is needed, without passing it as env var